pages/linux/vmstat.md

```diff
@@ -0,0 +1,8 @@
+# vmstat
+
+> Report information about processes, memory, paging, block IO, traps, disks and CPU activity.
+> More information: <https://linux.die.net/man/8/vmstat>.
+
+- Display reports every 2 seconds for 5 times:
```

Comment:
We should always start with the simplest invocation of the command, which is - `vmstat`. Would you mind sending another PR to have `vmstat` as the first example ?

Reply:
For future reference, that was done in #3927.